.github/workflows/build.yml

```diff
@@ -75,6 +75,36 @@ jobs:
             echo "run_hypothesis=true" >> $GITHUB_OUTPUT
           fi
 
+  check_abi:
+    name: 'Check if the ABI has changed'
+    runs-on: ubuntu-20.04
```

Comment:
Can we use?
```suggestion
    runs-on: ubuntu-latest
```
Or:
```suggestion
    runs-on: ubuntu-22.04
```

Reply:
Not latest, we need to pin the version here.